.github/workflows/pr_tools.yml

```
@@ -3,22 +3,22 @@ name: Test Tools
 on:
   push:
     branches:
-      - master
+      - main
   pull_request:
     branches:
-      - master
+      - main
 jobs:
   build:
 
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9]
```

These are all very old versions of Python. 3.8 is the oldest still supported, maybe test on it and the current 3.13?
```suggestion
        python-version: [3.8, 3.13]
```

---

I've selected 3.12.3